tests/unit/components/labeled-radio-button-test.js

```diff
@@ -38,6 +38,28 @@ test('it gives the label of a wrapped checkbox a `checked` className', function(
   assert.equal(this.$('label').hasClass('checked'), true);
 });
 
+test('it gives the label of a wrapped checkbox a user supplied `checked` className', function(assert) {
```

Comment:
perhaps something like 'specifying `checkedClass` changes the classname used on the label when the input is checked'